scripts/breaking_changes_checker/breaking_changes_tracker.py

```diff
@@ -504,6 +538,52 @@ def check_positional_parameter_added(self, current_parameters_node: Dict) -> Non
                     )
                 )
 
+    def check_non_positional_parameter_added(self, current_parameters_node: Dict) -> None:
+        if current_parameters_node["param_type"] != "positional_or_keyword":
+            if self.class_name:
+                if self.function_name == "__init__":
+                    # This is a new class property defined only through the init and not at the class level
+                    self.features_added.append(
+                        (
+                            self.ADDED_CLASS_PROPERTY_MSG, ChangeType.ADDED_CLASS_PROPERTY,
+                            self.module_name, self.class_name, self.parameter_name
+                        )
+                    )
+                else:
+                    self.features_added.append(
+                        (
+                            self.ADDED_CLASS_METHOD_PROPERTY_MSG, ChangeType.ADDED_CLASS_METHOD_PROPERTY,
+                            self.module_name, self.class_name, self.parameter_name, self.function_name
+                        )
+                    )
+            else:
+                self.features_added.append(
+                    (
+                        self.ADDED_FUNCTION_PARAMETER_MSG, ChangeType.ADDED_FUNCTION_PARAMETER,
+                        self.module_name, self.function_name, self.parameter_name
+                    )
+                )
+
+    def check_positional_parameter_added(self, current_parameters_node: Dict) -> None:
```

Comment:
this function seems to already be defined on L521 above - is there something changed here?

Reply:
Thanks for catching this, copy/paste error. Cleaning it up